### ISSUE TITLE
Update default ISI and NSF mappings

### ISSUE THREAD

**Issue by bherr2** [cns-iu/make-a-vis#122]
Use the attached file for updating the template projects.

[https://app.zenhub.com/files/142600324/e9e557b7-8086-4eb5-a4cc-ce6d28a6bcbb/download](https://app.zenhub.com/files/142600324/e9e557b7-8086-4eb5-a4cc-ce6d28a6bcbb/download)

**Comment by bherr2** [comment]
ISI has been updated to match the template provided in `develop`. 

A few notes:
- I used Discipline color for coloring the scattergraph which was one of the options.
- For tooltips I used the whole string since it seemed to look good without trying to concatenate it.

There were a couple questions on the template I can answer:
- What is that big black dot on top of scatter graph publications? A publication titled "Emergence of Scaling in Random Networks". Published in 1955 with over 900 citations.
- Where is full name in the record set? In the FourNetworkSci.isi file, there are not full names provided. Works with newer ISI files just fine, it probably just wasn't included/available when that ISI file was made.

Still left:
* Temporal Bar Graph related things
